bin/www

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
```

Comment:
I basically took the default `bin/www` that express generates and moved the listening on ports stuff over as-is in to the file

Reply:
Can I still use nodemon on the server?

Reply:
Yes, I'll verify again but this is how I typically set up my start script - 

```
  "start": "npm install && nodemon ./bin/www",
```

Reply:
you need to chmod the www file?

Reply:
I don't that I know of